src/renderer/components/watch-video-info/watch-video-info.scss

```diff
@@ -22,6 +22,17 @@
   gap: 10px;
 }
 
+.unlistedBadge {
+  background-color: var(--secondary-card-bg-color);
+  border-radius: 5px;
+  display: inline;
+  padding-inline: 5px;
```

Comment:
```suggestion
  padding: 2px 5px;
```
I found the badge too thin
![image](https://github.com/user-attachments/assets/bec3e4c0-8383-4380-9e46-fd25850ee349)